src/components/10-atoms/fieldset/index.js

```diff
@@ -0,0 +1,31 @@
+import '@webcomponents/webcomponentsjs';
```

Comment:
please remove this, sorry again i just fixed today this issue in scaffolding

Reply:
done